packages/test/stochastic-test-utils/src/markovChain.ts

```diff
@@ -0,0 +1,234 @@
+import { IRandom } from "./types";
+
+abstract class MarkovChain {
```

Comment:
@Abe27342 - FYI.  Sean is starting to look into adding MarkovChain-based generators into 'stochastic-test-utils'.  The primary motivation is to be able to random strings that yield similar compression ratios to written prose.  (In this case, we're generating random tweets.)

Reply:
Neat! I'd be happy to review once ready.